nmmo/task/base_predicates.py

```diff
@@ -87,7 +92,7 @@ def CanSeeGroup(gs: GameState,
                 target: Group               = constraint.TEAM_GROUPS):
   """ Returns True if subject can see any of target
   """
-  return OR(*(CanSeeAgent(subject, agent) for agent in target.agents))
+  return POR(*(CanSeeAgent(subject, agent) for agent in target.agents))
```

Comment:
why POR not OR?

Reply:
OR for Predicate. But I hear you, and am removing all P prefix.